autogen/agentchat/contrib/qdrant_retrieve_user_proxy_agent.py

```diff
@@ -150,21 +150,21 @@ def retrieve_docs(self, problem: str, n_results: int = 20, search_string: str =
 
 def create_qdrant_from_dir(
     dir_path: str,
-    max_tokens: int = 4000,
-    client: QdrantClient = None,
-    collection_name: str = "all-my-documents",
-    chunk_mode: str = "multi_lines",
-    must_break_at_empty_line: bool = True,
-    embedding_model: str = "BAAI/bge-small-en-v1.5",
-    custom_text_split_function: Callable = None,
-    custom_text_types: List[str] = TEXT_FORMATS,
-    recursive: bool = True,
-    extra_docs: bool = False,
-    parallel: int = 0,
-    on_disk: bool = False,
+    max_tokens: Optional[int] = 4000,
```

Comment:
Is it a good idea to use `Optional` when there is a default? @jackgerrits @davorrunje